test-app/package.json

```diff
@@ -80,15 +80,15 @@
     "ember-cli-inject-live-reload": "^2.1.0",
     "ember-disable-prototype-extensions": "1.1.3",
     "ember-load-initializers": "^2.1.2",
-    "ember-qunit": "^6.1.1",
+    "ember-qunit": "^6.2.0",
```

Comment:
We need to update this dependency as ember 5 canary has released and this version has support for it (see https://github.com/CrowdStrike/ember-toucan-core/pull/58 for more info).  Otherwise the canary tests will fail https://github.com/CrowdStrike/ember-toucan-styles/actions/runs/4474010740/jobs/7862041288